cas-server-support-saml-idp/src/main/java/org/jasig/cas/support/saml/web/idp/profile/builders/SamlProfileSamlNameIdBuilder.java

```diff
@@ -31,6 +31,8 @@
 public class SamlProfileSamlNameIdBuilder extends AbstractSaml20ObjectBuilder implements SamlProfileObjectBuilder<NameID> {
     private static final long serialVersionUID = -6231886395225437320L;
 
+    private static final String DEFAULT_NAME_ID_FORMAT = "urn:oasis:names:tc:SAML:2.0:nameid-format:transient";
```

Comment:
I think this should be changed to use `unspecified`. Also, I am somewhat sure opensaml has the same constant somewhere. could we see if we could use that instead?